functions/package.json

```diff
@@ -13,7 +13,7 @@
   "engines": {
     "node": "16"
   },
-  "main": "lib/index.js",
+  "main": "lib/functions/src/index.js",
```

Comment:
Updated this to fix error at: https://github.com/covid-projections/act-now-links-service/actions/runs/3240139116/jobs/5310396297